mRemoteNGTests/Connection/AbstractConnectionInfoDataTests.cs

```diff
@@ -405,6 +405,15 @@ public void MacAddressNotifiesOnValueChange()
 
         [Test]
         public void UserFieldNotifiesOnValueChange()
+        {
+            var wasCalled = false;
+            _testAbstractConnectionInfoData.PropertyChanged += (sender, args) => wasCalled = true;
+            _testAbstractConnectionInfoData.Favorite = true;
```

Comment:
I think you transposed these two tests when copying. The `UserFieldNotifiesOnValueChange` test is setting the `Favorite` setting, and the `FavoriteNotifiesOnValueChange` test is setting the `UserField` setting